portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,55 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script>
+
+        function addRandomFact() {
+            const facts = 
+                ['hair', 'height', 'lakers', 'hammock', 'ucsb', 'italian'];
+
+            // Pick a random fact.
+            const fact = facts[Math.floor(Math.random() * facts.length)];
+
+            // Add it to the page.
+            const showFact = document.getElementById(fact);
+            showFact.style.display = "block";
```

Comment:
Is the intention to replace the current random fact or to append a new one?  What happens when you've shown all the random fact or tries to show a random fact previously shown?  I think it will appear like a no-op since the div is already displayed.

Reply:
If you want to hide any visible fact before showing the randomly selected one and we assume the `visible` class name from the above comment, you could do
```
document.querySelectorAll('.fact.visible').forEach(element => element.classList.remove('visible'));
```
I have a tendency to get overly "fancy" trying to use these sorts of functional programming constructs, so in case this doesn't make sense, the idea is that it's using `document.querySelectorAll` to get a list of all elements that have both the `fact` and `visible` classes, and for each element in that list (if any), executes the little function specified by the `=>` "arrow" syntax. That function says that we'll call the current element that's being iterated over `element`, and uses it to get the list of the classes that the element currently has, then remove the `visible` class.

If you think that approach decreases readability or want to do it some other way for any reason, feel free to take any approach that you think makes sense, e.g. a traditional `for` loop, or plain `document.querySelector`, or whatever else floats your boat.